backend/src/schema/resolvers/notifications.spec.js

```diff
@@ -191,6 +217,36 @@ describe('given some notifications', () => {
             query({ query: notificationQuery, variables: { ...variables, read: false } }),
           ).resolves.toEqual(expected)
         })
+
+        describe('if a resource gets deleted', () => {
+          const deletePostAction = async () => {
+            authenticatedUser = await author.toJson()
+            const deletePostMutation = gql`
+              mutation($id: ID!) {
+                DeletePost(id: $id) {
+                  id
+                  deleted
+                }
+              }
+            `
+            await expect(
+              mutate({ mutation: deletePostMutation, variables: { id: 'p3' } }),
+            ).resolves.toMatchObject({ data: { DeletePost: { id: 'p3', deleted: true } } })
+            authenticatedUser = await user.toJson()
+          }
+
+          it('reduces notifications list', async () => {
+            await expect(
+              query({ query: notificationQuery, variables: { ...variables, read: false } }),
+            ).resolves.toMatchObject({
+              data: { notifications: [expect.any(Object), expect.any(Object)] },
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-03T09:43:55Z" title="Tuesday, September 3rd 2019, 11:43:55 am +02:00">Sep 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -191,6 +217,31 @@ describe('given some notifications', () => {
             query({ query: notificationQuery, variables: { ...variables, read: false } }),
           ).resolves.toEqual(expected)
         })
+
+        describe('if a resource gets deleted', () => {
+          beforeEach(async () => {
+            authenticatedUser = await author.toJson()
+            const deletePostMutation = gql`
+              mutation($id: ID!) {
+                DeletePost(id: $id) {
+                  id
+                  deleted
+                }
+              }
+            `
+            await expect(
+              mutate({ mutation: deletePostMutation, variables: { id: 'p3' } }),
+            ).resolves.toMatchObject({ data: { DeletePost: { id: 'p3', deleted: true } } })
+            authenticatedUser = await user.toJson()
+          })
+
+          it('reduces notifications list', async () => {
+            const expected = expect.objectContaining({ data: { notifications: [] } })
+            await expect(
+              query({ query: notificationQuery, variables: { ...variables, read: false } }),
+            ).resolves.toEqual(expected)
```


did you try to get this to pass without lines 232-234?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-03T13:41:18Z" title="Tuesday, September 3rd 2019, 3:41:18 pm +02:00">Sep 3, 2019</time>_

---


done